BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -107,6 +107,8 @@
 "ExtensionInSafari" = "In Safari, find Bitwarden using the share icon (hint: scroll to the right on the bottom row of the menu).";
 "ExtensionInstantAccess" = "Get instant access to your passwords!";
 "ExtensionReady" = "You're ready to log in!";
+"ExtensionAllSet" = "You're all set!";
+"ExtensionAutoFillActivated" = "You can now use autofill to log into apps and websites using your saved passwords. Now, you can explore everything else Bitwarden has to offer.";
```

Comment:
🎨 First one, update key to match the values; in the second one we can use the `DescriptionLong` suffix so we maintain the same approach as with other long strings.
```suggestion
"YouReAllSet" = "You're all set!";
"ExtensionAutoFillActivatedDescriptionLong" = "You can now use autofill to log into apps and websites using your saved passwords. Now, you can explore everything else Bitwarden has to offer.";
```

Reply:
sounds good.  I thought I was following the pattern here but I'll make the change you sggested. 

```
"ExtensionReenable" = "Reactivate app extension";
"ExtensionAlmostDone" = "Almost done!";
"ExtensionEnable" = "Activate app extension";
"ExtensionInSafari" = "In Safari, find Bitwarden using the share icon (hint: scroll to the right on the bottom row of the menu).";
"ExtensionInstantAccess" = "Get instant access to your passwords!";
"ExtensionReady" = "You're ready to log in!";
"ExtensionAllSet" = "You're all set!";
"ExtensionAutoFillActivated" = "You can now use autofill to log into apps and websites using your saved passwords. Now, you can explore everything else Bitwarden has to offer.";
"ExtensionSetup" = "Your logins are now easily accessible from Safari, Chrome, and other supported apps.";
"ExtensionSetup2" = "In Safari and Chrome, find Bitwarden using the share icon (hint: scroll to the right on the bottom row of the share menu).";
"ExtensionTapIcon" = "Tap the Bitwarden icon in the menu to launch the extension.";
"ExtensionTurnOn" = "To turn on Bitwarden in Safari and other apps, tap the \"more\" icon on the bottom row of the menu.";
```